bin/improver-weighted-blending

```diff
@@ -84,8 +85,9 @@ def main():
                              '"weighted_maximum": multiplies the values in the'
                              ' coordinate by the weights, and then takes the'
                              ' maximum.')
-    parser.add_argument('input_filepath', metavar='INPUT_FILE',
-                        help='A path to an input NetCDF file to be processed.')
+    parser.add_argument('input_filepaths', metavar='INPUT_FILES',
+                        nargs="+",
+                        help='Paths to input files to be blended.')
```

Comment:
Do we need to specify that the input files are NetCDF?, e.g. 
help='Paths to input NetCDF files to be blended.'

Same question would apply to other CLI tests that specify INPUT_FILES.

Reply:
No I don't think so, I have had this discussion with Ben F, the nice thing about Iris is that it can read in various file types. We are using NetCDF but other people who want to use improver might want to use a different input format. Although we haven't tested it tying down the Input files to one format seems unnecessary. Hence I deliberately left out NetCDF from the help.

Reply:
That makes sense :)